pkg/otlp/internal/serializerexporter/consumer.go

```diff
@@ -15,20 +15,46 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/quantile"
 	"github.com/DataDog/datadog-agent/pkg/serializer"
 	"github.com/DataDog/datadog-agent/pkg/tagger"
-	"github.com/DataDog/datadog-agent/pkg/tagset"
+	"github.com/DataDog/datadog-agent/pkg/tagger/collectors"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
 var _ translator.Consumer = (*serializerConsumer)(nil)
 
 type serializerConsumer struct {
-	series   metrics.Series
-	sketches metrics.SketchSeriesList
+	cardinality collectors.TagCardinality
+	series      metrics.Series
+	sketches    metrics.SketchSeriesList
+}
+
+// enrichedTags of a given dimension.
+// In the OTLP pipeline, 'contexts' are kept within the translator, and,
+// therefore, this works a little differently than for DogStatsD/check metrics.
+func (c *serializerConsumer) enrichedTags(dimensions *translator.Dimensions) []string {
+	enrichedTags := make([]string, 0, len(dimensions.Tags()))
+	enrichedTags = append(enrichedTags, dimensions.Tags()...)
+
+	entityTags, err := tagger.Tag(dimensions.OriginID(), c.cardinality)
+	if err != nil {
+		log.Tracef("Cannot get tags for entity %s: %s", dimensions.OriginID(), err)
+	} else {
+		enrichedTags = append(enrichedTags, entityTags...)
+	}
+
+	orchestratorTags, err := tagger.OrchestratorScopeTag()
+	if err != nil {
+		log.Error(err.Error())
```

Comment:
let's lower the log level since this is a hot path, it can be spammy. (debug or trace should be enough?)

Reply:
I copied it from here https://github.com/DataDog/datadog-agent/blob/2e45d3acf10f9a3ca53689b92c8dde23fdb49aa5/pkg/tagger/global.go#L266 but agree with you that trace is safer. Changed!